src/main/java/core/basesyntax/Application.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.lang.reflect.Array;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public class Application {
+    FigureSupplier figureSupplier = new FigureSupplier();
+    public static void main(String[] qwe) {
```

Comment:
The parameter name 'qwe' is not meaningful. Use a name that reflects its purpose, such as 'args'.